bip39-lib/src/jvmMain/kotlin/cash/z/ecc/android/random/SecureRandom.kt

```diff
@@ -0,0 +1,9 @@
+package cash.z.ecc.android.random
+
+actual class SecureRandom actual constructor() {
+    val secureRandom = java.security.SecureRandom()
```

Comment:
On native I'm getting nextBytes with read `/dev/urandom` [which won't block](https://linux.die.net/man/4/urandom) so suspend shouldn't be needed. and `BCryptGenRandom` on windows (but I can't find info on whether or not that could block)

On `JS` I'm currently using `crypto.getRandomValues(bytes)` which isn't a `promise`. so `suspend` isn't needed there.
But if you think `generateKey` [needs to be used instead 
](https://developer.mozilla.org/en-US/docs/Web/API/Crypto/getRandomValues#usage_notes), that returns a promise. Which means that function would need to be a suspend

and for jvm: it looks like it could technically block not the thread ... or it could https://stackoverflow.com/questions/137212/how-to-deal-with-a-slow-securerandom-generator

If it were me. I wouldn't make it a suspend if it doesn't need to be

Reply:
Or both suspending and non-suspending versions could be made. Or keep none of them suspending and make a library level suspending initialize function similar to https://github.com/ionspin/kotlin-multiplatform-libsodium

Reply:
Thanks for the input—let's leave this as-is for now.  It is something to keep incubating on as we move forward.

What I've been doing to date is wrapping the usage in a [withContext()](https://github.com/zcash/secant-android-wallet/blob/4f144405c148b7a60d6164777bad580b921dbaf1/sdk-ext-lib/src/main/java/cash/z/ecc/sdk/model/SeedPhrase.kt#L21) which works fine for the time being.  I like the idea of a suspending public API eventually because it better signals to clients that the call might be slower.